context.go

```diff
@@ -11,10 +12,11 @@ var (
 
 var _ context.Context = &Context{}
 
-// A Context is the default routing context set on the root node of a
+// Context is the default routing context set on the root node of a
 // request context to track URL parameters and an optional routing path.
 type Context struct {
-	context.Context
+	// Parent context
+	parent context.Context
```

Comment:
this will make Context unexported. 
the context implementations of the standard library has Context exported and this will also break compatibility with chi v1. For example:
c := chi.Context{}
c.Context = context.Background()
will not compile.